orchestrator-salt/src/main/resources/salt/salt/cloudera/csd/csd-downloader.sh

```diff
@@ -3,6 +3,7 @@
 set -ex
 
 mkdir -p /opt/cloudera/csd
+rm -f /opt/cloudera/csd/*
 cd /opt/cloudera/csd
 
 {% if salt['pillar.get']('cloudera-manager:csd-urls') %}
```

Comment:
There should be a test -f within the loop to skip the download if we already have it.